bin/varnishd/cache/cache_vrt.c

```diff
@@ -966,17 +966,20 @@ VRT_blob(VRT_CTX, const char *err, const void *src, size_t len, unsigned type)
 	return (p);
 }
 
+static const struct vrt_blob null_string_blob = {
+	.blob = "(null)",
+	.len = sizeof "(null)" - 1,
+};
+
 VCL_BLOB
-VPI_Blob(struct vrt_blob *b, const char *s)
+VPI_blob(VRT_CTX, VCL_STRING s)
 {
 
 	if (s == NULL)
-		s = "(null)";
+		return (&null_string_blob);
```

Comment:
we should move this to VRT_blob()

Reply:
VRT_blob() doesn't take a STRING argument.

Reply:
oh sorry, yes. /me bad